.travis.yml

```diff
@@ -19,6 +24,9 @@ after_success:
   - yarn global add travis-deploy-once@4
   - travis-deploy-once "yarn global add semantic-release@12 && semantic-release"
 
+notifications:
```

Comment:
Please remove this directive. It is unnecessary. We don't receive notifications unless there was a failure which is useful.

Reply:
It was already present in your file.

Reply:
Ah.. I see. My bad.